.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+# Rule definitions: http://flake8.pycqa.org/en/latest/user/error-codes.html
+# D203: 1 blank line required before class docstring
+# F401: 'identifier' imported but unused
+# E402: module level import not at top of file
+# W503: line break before binary operator
+exclude = venv*,__pycache__,node_modules,bower_components
+ignore = D203,W503
```

Comment:
Would be nice if W503 appeared in the definitions in the same order it appeared here (i.e. second in the list), but eh.

Reply:
I'm envisaging that comment section more like an alphabetical index section in a book rather than a contents section (except, not alphabetical, numerical 😁 ).

My reasoning behind this is that similar rules are grouped by number eg.
```
E701 | multiple statements on one line (colon)
E702 | multiple statements on one line (semicolon)
```
And are listed in order in these 2 lists:
https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes
http://flake8.pycqa.org/en/latest/user/error-codes.html


I'd have to apply this change to the other apps for consistency if we do want to reorder them.